Library/Homebrew/cmake/trap_fetchcontent_provider.cmake

```diff
@@ -1,4 +1,6 @@
-cmake_minimum_required(VERSION 3.24) # Dependency providers introduced in CMake 3.24
```

Comment:
Yea, it seems setting this should be done by the project and not downstream consumers so we shouldn't have set this in the first place.

We could maybe still check the version of CMake and error out saying they should `brew upgrade cmake` if their CMake is too old though.